### DISCUSSION TITLE
Grow the community

### DISCUSSION THREAD

**Discussion by tcharding** [rust-bitcoin/rust-bitcoin#1057] [General]
Totally non-urgent.

Hey @apoelstra, can we add folks to the `rust-bitcoin` github organisation please? I'm not exactly sure of a good metric to use for membership but the current 5 members seems a bit little for how much work is going on.

Cheers man.

**Comment by apoelstra** [comment]
Sure, I'll add anybody who posts here and who I recognize and trust enough to let them create new repos.

Added you.

**Reply by tcharding** [reply]
Thanks, feels good.

**Comment by Kixunil** [comment]
Is the idea that if one creates any MSRV 1.41-compliant bitcoin crate he can add it here to get more visibility?

**Reply by apoelstra** [reply]
I think we should come up with a couple more guidelines ... but essentially yes, that sounds good to me.

**Reply by Kixunil** [reply]
Then I'm interested, would like to move some crates here (eventually). :)

**Reply by apoelstra** [reply]
Cool :) I've invited you to the community. But we should have a separate discussion about what crates can come under this org.

**Comment by sanket1729** [comment]
I would like an invite here. I don't want to add repos in the near future, but having the recognition feels good :)

**Reply by dpc** [reply]
Oh ya. Now that you mention it, I made a PR once or twice, so I think I deserve to flex!

(I initially though the whole thing involves some effort and responsibility which generally are not my favorite things, but if I could only be a figurehead and make the community stats looks better, I very much would like to do so)

**Reply by apoelstra** [reply]
Thanks for the humility, but new contributors should also know that @dpc originally created rust-secp256k1 and handed it off to me once rust-bitcoin had gotten going :)

I'll send an invite to both of you.

**Reply by tcharding** [reply]
Lolz so many laughs today - I love it. Coincidentally I only noticed this week that @dpc was a [dark horse](https://dictionary.cambridge.org/dictionary/english/dark-horse) when I saw in `rust-secp256k1`:
```
authors = [ "Dawid Ciężarkiewicz <dpc@ucore.info>",
            "Andrew Poelstra <apoelstra@wpsoftware.net>" ]
```
